tests/test_cases/forkandopen.c

```diff
@@ -24,7 +24,7 @@ int main()
     {
         printf("Parent: Testing open in parent: pid %d!\n", pid); 
 
-        int fd = open("foo.txt", O_RDONLY | O_CREAT);
+        int fd = open("foo.txt", O_RDONLY|O_CREAT);
```

Comment:
In general, spaces between operand and operator is what we've been going for in lind, and these changes should not be necessary